src/defaultConfig/README.md

```diff
@@ -27,7 +27,7 @@ Object containing options to pass through to KSS-node. I can't find a good sourc
 * **prototypes** {array} : default `['index']` - Array of prototypes to generate via HTML webpack plugin. For each array entry you can either pass in a single string corresponding to the title of the prototype, or an object containing option overrides for HTML webpack plugin [(configuration)](https://github.com/ampedandwired/html-webpack-plugin). If you use an object, you must at least provide a `title` field. The title field, whether passed in as a string or the `title` property value, must be the same name as your prototype file located in your `prototypes` directory. So for example, if you provided `['homepage']` in the prototypes option, you would need to have a `prototypes/prototype-homepage.html` file in order for the prototype to display correctly.
 * **root** {string} : default `'dist/'` - Root directory for `webpack-dev-server`. All static assets you need for your prototype(s) should be located in this directory. The `output` option should be relative to this path.
 * **sectionTemplate** {string} : default `path.join(__dirname, '../templates/section.hbs')` - Override for the template used to produce styleguide sections. This should always be a handlebars file, and will utilize the KSS data as its source.
-* **classNames** {string} : For use with CSS Modules. This option should point to a directory with JSON files containing localized classnames. NOTE: at the moment this will only work a CSS Modules implementation that produces JSON output, like `postcss-modules`. The CSS modules implementation offered by the `modules` option in `css-loader` currently will not work.
+* **classNames** {string} : For use with CSS Modules. This option should point to a filepath with JSON files containing localized classnames. NOTE: at the moment this will only work a CSS Modules implementation that produces JSON output, like `postcss-modules`. The CSS modules implementation offered by the `modules` option in `css-loader` currently will not work.
```

Comment:
Change to: `For use with CSS Modules. This option should point to a JSON files containing localized classnames...` etc.